src/convertToObject.js

```diff
@@ -3,16 +3,32 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringToArray = sourceString.split(';');
+  const arrayWithinArray = stringToArray.map(v => v.split(':'));
+  const result = {};
+
+  for (const array of arrayWithinArray) {
+    const rawKey = array[0].split('\n');
```

Comment:
please consider trimming array
```suggestion
    const rawKey = array[0].trim();
```